src/tabs/styles/tabs-header.scss

```diff
@@ -0,0 +1,3 @@
+.fa.fa-check {
```

Comment:
gaat ge hierdoor niet alle checks in gans je aplicatie groen maken?

Reply:
Nee want die tabs-header scss wordt enkel geimplement in de template van de component zelf

Reply:
ah check, daardoor is die scope beperkt in de component ofwa?

Reply:
yes